external-crates/move/tools/move-cli/tests/build_tests/build_with_dep_warnings/dep/sources/has_warning.move

```diff
@@ -0,0 +1,5 @@
+module 0x42::m {
+public fun foo(x: u64): u64 {
```

Comment:
Nit. Is there a reason why these are not indented?

Reply:
Because I `cp`d the file and it didn't have them. Happy to change though